conbench/app/_plots.py

```diff
@@ -35,22 +38,19 @@ def get_outliers(self, benchmarks):
         return outliers, outlier_ids, outlier_names
 
     def get_history_plot(self, benchmark, run, i=0):
-        history = self._get_history(benchmark)
-        if history:
+        samples = get_history_for_benchmark(benchmark_result_id=benchmark["id"])
```

Comment:
I changed the name of this function and the name of the first argument to make this more readable.

Seeing this now, I consider changing this in the future to something like

```python
history_items = get_history_for_benchmark_result(result_id)
```